packages/query-persist-client-core/src/persist.ts

```diff
@@ -52,6 +53,20 @@ export interface PersistQueryClientOptions
     PersistedQueryClientSaveOptions,
     PersistQueryClienRootOptions {}
 
+/**
+ * Checks if emitted event is about cache change and not about observers.
+ * Useful for persist, where we only want to trigger save when cache is changed.
+ */
+function isCacheableEventType(eventType: NotifyEventType) {
+  const CACHEABLE_EVENT_TYPES: Array<NotifyEventType> = [
+    'added',
+    'removed',
+    'updated',
+  ]
```

Comment:
could we move this out of the `function isCacheableEventType` ? There's no need to re-create this every time.